### DISCUSSION TITLE
Add depend/if statements in transactional emails bassed on config

### DISCUSSION THREAD

**Discussion by seansan** [OpenMage/magento-lts#2109] [Ideas]
Feature request: add depend/if statements in transactional emails bassed on config

It would be great if we can use something like `{{depend {{config path='some/value/here'}}}} text here `

Step 1 would be to support true/false `{{depend {{config path='some/value/here'}}}}`

Step 2 would be to support equals == value `{{depend {{config path='some/value/here'}} == 'myvalue'}}`

Explained here

https://magento.stackexchange.com/questions/6507/magento-depend-combined-with-config-path-some-value-here

And possibly needs a change here: 

https://github.com/OpenMage/magento-lts/blob/f78f0c2421555d3c3f0f18f17a86cdd2a7315972/lib/Varien/Filter/Template.php#L131

**Comment by bluec** [comment]
At first this seems nice to extend or enhance the logic comparison available in the email templates, but it's so simple (a tiny bit of layout xml) to just breakout of the email template file and use a .phtml block instead which can then include whatever logic you wish, so is this really necessary?

**Comment by luigifab** [comment]
Sorry for this "ad" like, but [I did that for one of my module](https://github.com/luigifab/openmage-maillog/blob/master/src/app/code/community/Luigifab/Maillog/Model/Filter.php).
I also added support for:
```
{{if something gt/gte/gteq/lt/lte/lteq/eq/neq something/empty}} ... {{elseif ...}} ... {{else}} ... {{/if}}
{{if something in/nin a,b,c,1,2,3}} ... {{elseif ...}} ... {{else}} ... {{/if}} (in_array)
{{if something ct/nct something}} ... {{elseif ...}} ... {{else}} ... {{/if}} (contains)
{{ifconfig path='a/b/c'}} ... {{elseconfig}} ... {{/ifconfig}}
{{foreach something}} ... {{forelse}} ... {{/foreach}}
{{dump something}} | {{dump}}
{{number something}} | {{number config='a/b/c'}}
{{price something}} | {{price config='a/b/c'}}
```

**Comment by seansan** [comment]
wow! Would even be a great contribution to M2 too ;)
